.github/workflows/ci.yml

```diff
@@ -124,7 +124,9 @@ jobs:
         env:
           PHP_CS_FIXER_IGNORE_ENV: ${{ matrix.PHP_CS_FIXER_IGNORE_ENV }}
           FAST_LINT_TEST_CASES: ${{ matrix.FAST_LINT_TEST_CASES }}
-        run: vendor/bin/phpunit ${{ matrix.phpunit-flags }}
+        run: |
+          php -v
```

Comment:
@SpacePossum 
why we need this?
isn't it already part of the workflow, to set up configured version of PHP?

Reply:
-> #6172

Reply:
yeah if test locally work but no gitlab it is very much handly to know the patch version (like 8.0.0 vs. 8.0.1)